src/utils/constants.js

```diff
@@ -29,4 +29,12 @@ export const SINGULARS = {
 	transactions: 'transaction',
 };
 
-export const CONFIG_VARIABLES = ['json', 'name', 'pretty', 'testnet'];
+export const CONFIG_VARIABLES = [
```

Comment:
Maybe we should be generating this directly from the default config file now? Alternatively we could be specifying the types here, rather than in the `set` command file.

Reply:
What do you think about specifying the types here @Tosch110?

Reply:
@willclarktech what do you mean by "specifying the types"?

Reply:
In the `set` command we currently map these config variables to type handlers. But the relevant type could be specified already in `constants`.